bundle/edu.gemini.dataman.app/src/main/scala/edu/gemini/dataman/core/package.scala

```diff
@@ -33,7 +36,10 @@ package object core {
     def fail(f: => DmanFailure): DmanAction[Nothing] = EitherT(Task.delay(f.left))
 
     def mergeFailure[A](result: Throwable \/ TryDman[A]): TryDman[A] =
-      result.fold(ex => DmanFailure.DmanException(ex).left, identity)
+      result.fold({
+        case ie: InterruptedException => throw ie
```

Comment:
Wouldn't you rather return a left with the exception? Or you want here to kill the thread?